README.md

```diff
@@ -1 +1,45 @@
-# typescript-node-starter
\ No newline at end of file
+# Bulk Data Utilities 
+
+A library to handle  bulk import requests and interact with the [DEQM-Test-Server](https://github.com/projecttacoma/deqm-test-server)
```

Comment:
I'd link to the import ping and pull spec somewhere in here:

https://github.com/smart-on-fhir/bulk-import/blob/master/import-pnp.md